### DISCUSSION TITLE
Buttons für LONG und SHORT PRESS

### DISCUSSION THREAD

**Discussion by itn3rd77** [SukramJ/aiohomematic#238] [Ideas]
Hi Markus,

wäre es möglich [Buttons](https://www.home-assistant.io/integrations/button/) für die Geräte zu erzeugen die LONG und SHORT PRESS unterstützen?

Ich habe an den meisten Schaltern (HmIP-BSM) immer Sonderfunktionen für z.B. einen LONG PRESS hinterlegt. Das neue Button Entity wäre ja genau das richtige für so eine Funktion, dass läst sich nur leider nicht manuell erzeugen.

Vorteil von Buttons wäre auch, dass diese in der Kopfzeile einer Entities Card eingebunden werden können.

![grafik](https://user-images.githubusercontent.com/2655169/151310112-8d9888bb-8295-4e09-9275-57c26a3a37b5.png)

Damit hat man dann neben dem An/Aus auch gleich die zusätzlichen Funktionalitäten direkt an der Hand.

Danke und Gruß
Ingo

**Comment by danielperna84** [comment]
Passend dazu gibt es diese Diskussion: https://github.com/danielperna84/hahomematic/discussions/215
